src/components/PostInfo/PostInfo.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Post } from '../../types/Post';
+import { CommentList } from '../CommentList/CommentList';
+import { UserInfo } from '../UserInfo/UserInfo';
+
+type Props = {
+  post: Post
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => (
+  <>
+    <div className="posts__user">
+      {post.user && <UserInfo user={post.user} />}
+    </div>
```

Comment:
Maybe better don't render empty div if the post doesn't have a writer? And should comments fix too